corehq/apps/reports/tests/test_tableau_api_session.py

```diff
@@ -209,7 +220,7 @@ def failure_response(self):
         return mock_response
 
 
-class TestTableauAPI(TestCase):
+class TestTableauAPISession(TestCase):
```

Comment:
I'm wondering about the utility of these tests in their current form and whether they wouldn't be better if we used [requests-mock](https://requests-mock.readthedocs.io/en/latest/mocker.html) instead of normal mocking. The benefit is that using requests-mock would also validate the request method, path and params that we expect.

Probably not worth it for this PR but worth considering.

Reply:
I generally don't think integration tests with external systems belong in this build however it would be possible to set them up to run on a daily basis. Credentials can be added using GitHub secrets.